ci/release/changelogs/next.md

```diff
@@ -1,5 +1,5 @@
 #### Features 🚀
-
+- Circle notation is now supported. [#634](https://github.com/terrastruct/d2/pull/634)
```

Comment:
`Circle arrowheads are now supported`?

Reply:
I was just following the crows foot notation PR which was accepted.  I agree here also this is ambiguous at best.

Reply:
Reworded